sdk/storage/inc/blobs/internal/blob_client_options.hpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+#include <limits>
+#include <utility>
+
+#include "protocol/blob_rest_client.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+  struct BlobClientOptions
+  {
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+  };
+
+  struct BlockBlobClientOptions : public BlobClientOptions
+  {
+
+  };
+
+  struct GetBlobPropertiesOptions
+  {
+    Azure::Core::Context Context;
+  };
+
+  struct SetBlobHttpHeadersOptions
```

Comment:
TODO: add comments for each parameter.

Reply:
do you mean the comments used by doxygen to generate documentation?